apps/meteor/app/custom-sounds/server/methods/insertOrUpdateSound.js

```diff
@@ -1,19 +1,19 @@
 import { Meteor } from 'meteor/meteor';
-import s from 'underscore.string';
 import { check } from 'meteor/check';
 import { CustomSounds } from '@rocket.chat/models';
 import { api } from '@rocket.chat/core-services';
 
 import { hasPermission } from '../../../authorization/server';
 import { RocketChatFileCustomSoundsInstance } from '../startup/custom-sounds';
+import { trim } from '../../../../lib/utils/stringUtils';
 
 Meteor.methods({
 	async insertOrUpdateSound(soundData) {
 		if (!hasPermission(this.userId, 'manage-sounds')) {
 			throw new Meteor.Error('not_authorized');
 		}
 
-		if (!s.trim(soundData.name)) {
+		if (!trim(soundData.name)) {
```

Comment:
soundData.name?.trim() can't be used?

Reply:
This `trim` implementation also handles the case of incorrect value types (e.g. `123`), coercing as needed. There is some risk.